src/components/Text.js

```diff
@@ -14,31 +14,21 @@ type Props = {
   markdown?: boolean,
   style?: StyleObj,
   allowFontScaling?: boolean,
-  onLayout?: Function
+  onLayout?: Function,
+  numberOfLines?: number
 };
 
-const Text = ({
-  children,
-  type,
-  markdown,
-  style,
-  allowFontScaling,
-  onLayout
-}: Props) =>
+const Text = ({ children, type, markdown, style, ...otherProps }: Props) =>
   markdown ? (
     <Markdown
       style={style}
       markdownStyles={{ ...textStyles, ...markdownStyles }}
-      onLayout={onLayout}
+      {...otherProps}
```

Comment:
We might get warnings passing all the props to the markdown element. From their source code it looks like they pass down any extra props to a `View` rather than `Text`.

I hadn't noticed earlier that this makes things a bit more tricky. Because some props like `onLayout` are accepted by `View`, but others like `allowFontScaling` won't be. If the dev build runs fine without warnings then this is fine I think. Otherwise we might have to be explicit just with this component like:

```
<Markdown
  style={style}
  markdownStyles={{ ...textStyles, ...markdownStyles }}
  onLayout={otherProps.onLayout}
>
```
etc

Reply:
debated this one with @frigus02 and wasn't sure so happy to revert back

Reply:
Yeah, we were not sure about this. It might be a bit confusing. But looks like flow is fine with it.

Reply:
I dunno really. If there aren't any warnings in the console, then I think it's all good as you've got it

Reply:
I remember at some point React added warnings when unknown props were passed to a component, but that might not be the case anymore.